feat(xmllib): enable to save warning messages in csv through an .env configuration (DEV-4853) (#1692)
dasch-swiss/dsp-tools: docs/xmllib-api-reference/advanced-set-up.md

# Advanced Set-Up Options

> ## Save User Information to CSV

Reviewer: ```suggestion
## Save Output to CSV
```

User information, warnings and errors are printed out on the command line.

> To save them to a csv file, you can set that up through a `.env` file through the following steps:

Reviewer: ```suggestion
To additionally save them to a CSV file, you can set that up through a `.env` file through the following steps:
```

> - In the directory from where you run your code create a new file called `.env`. 
>   If you already have a `.env` file you can add the new line directly.
> - Enter the path of the csv file in quotation marks with the variable `WARNING_CSV_SAVEPATH`

Reviewer: ```suggestion
- In the directory from where you run your code, create a new file called `.env`. 
  If this file already exists, continue with the next step. 
- Set `WARNING_CSV_SAVEPATH` to the path of the CSV file
```

Example `.env` file content:

  ```env

>   WARNING_CSV_SAVEPATH="my_folder/my_file.csv"

Reviewer: I suggest calling it WARNINGS_CSV_SAVEPATH (plural)
